src/shared/store/configure-store.js

```diff
@@ -13,6 +16,13 @@ export default function configureStore(initialState, scope = 'main') {
   let middleware = [thunk];
 
   if (scope === 'renderer' && process.env.NODE_ENV === 'development') {
+    whyDidYouRender(React, {
+      include: [/^Connect/],
+      logOnDifferentValues: true,
+      trackHooks: true,
+      collapseGroups: true
+    });
```

Comment:
Hmm, I wonder if this should only be enabled via a flag. We probably don't want it running 100% of the time - Probably only when we want to do specific debugging relating to multiple component renders. @sallar?